kernel/src/hil/acifc.rs

```diff
@@ -0,0 +1,40 @@
+//! Interface for direct control of ACIFC
+//!
+//! Author: Danilo Verhaert <verhaert@cs.stanford.edu>
+
+use returncode::ReturnCode;
+
+/// Enum for selecting which edge to trigger interrupts on.
+// #[derive(Debug)]
+// pub enum InterruptMode {
+//     RisingEdge,
+//     FallingEdge,
+//     EitherEdge,
+// }
+
+pub trait Acifc {
+    /// Initialize the ACIFC
+    fn initialize_acifc(&self) -> ReturnCode;
+
+    /// Enable the clock
+    fn enable_clock(&self);
+
+    /// Disable the clock
+    fn disable_clock(&self);
+
+    /// Do a comparison of two inputs. Output will be 1 when Vinp>Vinn (Vin positive > Vin negative)
+    fn normal_comparison(&self, usize) -> u32;
```

Comment:
Maybe a more informative name than `normal_`... specifically this method returns whether Vinp > Vinn, so the name should just say that. What about just "compare"?

What is the `usize` argument?

What about a more specific return type? Either a `bool` (if the return value is really only going to be `1` or `!1`. Alternatively, an enum that values only in the possible range.

Reply:
Returning as bool now. However, when using ReturnCode::SuccessWithValue a usize is expected, so in the capsule it's converted to a usize anyways.